### DISCUSSION TITLE
Is rust-analyzer the one generating the `for` suggestion?

### DISCUSSION THREAD

**Discussion by jmg-duarte** [rust-lang/rust-analyzer#17237] [Q&A] [answered]
This one to be precise:
<img width="939" alt="Screenshot 2024-05-15 at 10 29 33" src="https://github.com/rust-lang/rust-analyzer/assets/15343819/ad262329-880a-48f1-9a3e-98fa8369a77f">

I wanted to do something similar for `print` and I'm not sure if it is provided by RA or VSCode

**Comment by davidbarsky** [comment, marked as answer]
It's defined in rust-analyzer: https://github.com/rust-lang/rust-analyzer/blob/master/crates/ide-completion/src/completions/postfix.rs#L141
